package.json

```diff
@@ -81,6 +84,6 @@
   "packageManager": "pnpm@8.6.10",
   "volta": {
```

Comment:
For anyone who might be curious how volta respects (or doesn't for now) this `packageManager` field, I found these issues with some context:
- https://github.com/volta-cli/volta/issues/987
- https://github.com/volta-cli/volta/issues/282